src/web/Animated.tsx

```diff
@@ -149,32 +149,32 @@ export class Value extends Types.AnimatedValue {
     }
 
     // Add listener for when the value gets updated.
-    addListener(listenerToAdd: ValueListener): void {
+    _addListener(listenerToAdd: ValueListener): void {
```

Comment:
Shouldn't this (and the rest) have the private modifier? They're still exposed as public on ValueListener

Reply:
That won't work because these methods on Value need to be callable from within the AnimatedComponentGenerated class. In other words, they need to have public accessibility within reactxp's implementation but not part of its public interface. Unfortunately, TypeScript doesn't have any way of expressing this.